js/views/media_overlay_data_injector.js

```diff
@@ -31,15 +31,16 @@ define (["jquery", "underscore", "../helpers", "../models/smil_iterator", "rangy
  */
 var MediaOverlayDataInjector = function (mediaOverlay, mediaOverlayPlayer) {
 
-    this.attachMediaOverlayData = function ($iframe, spineItem, mediaOverlaySettings) {
-
+    this.attachMediaOverlayData = function ($iframe, spineItem, mediaOverlaySettings, tapEmitter) {
         var contentDocElement = $iframe[0].contentDocument.documentElement;
+        var $body = $("body", contentDocElement);
 
         if (!spineItem.media_overlay_id && mediaOverlay.smil_models.length === 0) {
+            if ($body.length > 0) {
+                Helpers.addTapEventHandler($body[0], tapEmitter);
+            }
             return;
```

Comment:
Thank you for this contribution. Touch handling improvements in the Media Overlay implementation are welcome indeed.